internal/flag/context.go

```diff
@@ -42,23 +41,29 @@ func FirstArg(ctx context.Context) string {
 	return ""
 }
 
-func CmdParentName(cmd *cobra.Command) cobra.Command {
-	parent := cmd.Parent()
-	if strings.ToUpper(parent.Name()) == "FLY" {
-		return *cmd
+func EnvNameFromCmd(cmd *cobra.Command) string {
+	if cmd.Parent() != nil {
```

Comment:
Does this break existing users of FLY_ORG?

Reply:
No, I broke that elsewhere; this is the "fix", but with a caveat:
`FLY_ORG` by itself as an env var will do what you expect, but if you for instance do `FLY_LAUNCH_ORG`, that will get precedence. Some examples:

list apps for org xxx:
```sh
FLY_ORG=xxx fly apps list 
```

list apps for org yyy:
```sh
FLY_APPS_ORG=yyy fly apps list
```

list apps for org yyy:
```sh
FLY_ORG=xxx FLY_APPS_ORG=yyy fly apps list
```